Cargo.toml

```diff
@@ -7,3 +7,6 @@ repository = "https://github.com/dragostis/pest"
 documentation = "http://dragostis.github.io/pest/pest/"
 keywords = ["pest", "parser", "peg", "grammar"]
 license = "MPL-2.0"
+
+[dependencies]
+clap = "2.8.0"
```

Comment:
Good point. One of the few things that I tried to keep was 0 dependencies. Still, I don't know if features will do the trick here because you can't really do conditional compilation for logic and I need argument parsing in the tool.

Binary dependencies would be great, but they're not yet implemented in Cargo.

If you have a good idea how to do this, please leave a comment.